core/src/main/java/org/akaza/openclinica/service/pmanage/RandomizationRegistrar.java

```diff
@@ -57,10 +57,7 @@ public SeRandomizationDTO getRandomizationDTOObject(String studyOid) {
             }
 
         } catch (Exception e) {
-            logger.error(e.getMessage());
-            logger.error(ExceptionUtils.getStackTrace(e));
-            System.out.println(e.getMessage());
-            System.out.println(ExceptionUtils.getStackTrace(e));
+            logger.error("Response in not getting generated properly: {} ",e.getMessage(),e);
```

Comment:
you can just do: logger.error("Response in not getting generated properly", e);

Reply:
I have changed according to your comments